README.md

```diff
@@ -24,3 +24,32 @@ The connector is created automatically when the application is started. See the
 for a list of all available properties.
 
 The connector can be configured using a configuration file. See `localdev` for an example configuration file.
+
+# Restore-CLI
+
+To restore records from a backup run the program 
+
+```bash
+
+./gradlew :cli:run restore --bucket <s3-backup-bucket> --pattern <topicPattern>
+
+```
+
+The above command restores all records for a given topic to the same topic name.
+
+### All options:
+| Option name        | Short option | Required                                    | Format              | Description                                                                                                                      |
+|--------------------|--------------|---------------------------------------------|---------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| bucket             | b            | always                                      | String              | Bucket in which the backup is stored                                                                                             |
+| s3Endpoint         |              | If not restoring from AWS                   | Url                 | Endpoint for S3 backup storage                                                                                                   |
+| bootstrapServers   |              | If env `KAFKA_BOOTSTRAP_SERVERS` is not set | (list of) Urls      | Kafka cluster to restore the backup to                                                                                           |
+| awsAccessKeyId     |              | If env `AWS_ACCESS_KEY_ID` is not set       | String              | Access key id for s3                                                                                                             |
```

Comment:
don't allow to set aws credentials as arguments. However, you could think about adding a `--profile` parameter to use a proper aws profile if needed.